install.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-MBGLVERSION='3.4.1'
+MBGLVERSION='3.4.2'
```

Comment:
Yeah, d42768cff5e817d94f6d87ecd0d1372972d127e7 in the `release-v3.5.0` should have covered this.